packages/framework/src/hooks/useMockResponse.ts

```diff
@@ -0,0 +1,13 @@
+const mockResponseTable: Record<string, boolean> = {};
+
+export const useMockResponse = (appId: string | undefined) => {
+    if (typeof appId == 'undefined' || !(appId in mockResponseTable))
+        return false;
+    return mockResponseTable[appId];
+}
+
+export const setUseMockResponse = (appId: string | undefined, value: boolean) => {
+    if (typeof appId == 'undefined')
+        return;
+    mockResponseTable[appId] = value
+};
```

Comment:
Not sure if these functions have much value - you should be able to modify the object directly.